addon/templates/components/md-nav-link.hbs

```diff
@@ -0,0 +1,6 @@
+<a href={{href}}>
+{{#if icon}}
+  <i class="material-icons {{iconClass}}">{{icon}}</i>
+{{/if}}
+{{#if linkTitle}}{{linkTitle}}{{else}}{{yield}}{{/if}}
```

Comment:
The API for nav-link seems awkward if I want an icon only—have to specify a blank title:

```hbs
{{md-nav-link "" "index" icon="star"}}
```